fake_switches/terminal/telnet.py

```diff
@@ -60,14 +62,14 @@ def applicationDataReceived(self, data):
                     self.write(self._replace_input)
 
     def write(self, data):
-        self.transport.write(lf_to_crlf(data))
+        self.transport.write(lf_to_crlf(data).encode())
 
     def writeln(self, data):
         self.write(data)
         self.next_line()
 
     def next_line(self):
-        self.write(CR + LF)
+        self.write('\n')
```

Comment:
As you can see, lf_to_crlf() should handle this case so I think it is a good thing to use '\n' here.